e2e/suite1/specs/queryVariableCrud.spec.ts

```diff
@@ -1,74 +1,173 @@
 import { e2e } from '@grafana/e2e';
+import { describe } from '../../../public/test/lib/common';
```

Comment:
question: do we want to have a dependency in this direction? from this package on to the grafana project?

Reply:
> question: do we want to have a dependency in this direction? from this package on to the grafana project?

Exceptional catch, no we do not. I will fix! 🏅